bib/french.bib

```diff
@@ -4946,8 +4946,8 @@ @BOOK(
 )
 
 @MVBOOK(
-	mercenne_1636,
-	author = {F. Marin Mercenne},
+	mersenne_1636,
```

Comment:
Yes. I do. Redirect is not that hard to set up.
On Sun, 17 Jan 2016 at 11:08, Yuriy Chernyshov <notifications@github.com>
wrote:

> In bib/french.bib
> <https://github.com/georgthegreat/dancebooks-bibtex/pull/47#discussion_r49942990>
> :
>
> > @@ -4946,8 +4946,8 @@ @BOOK(
> >  )
> >
> >  @MVBOOK(
> > -	mercenne_1636,
> > -	author = {F. Marin Mercenne},
> > +	mersenne_1636,
>
> Do you consider changind id as really required measure?
> This would require adding a redirection in all the four configs
> (configs/config.*
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/georgthegreat/dancebooks-bibtex/pull/47/files#r49942990>
> .
>